jsonschema-module-jackson/src/test/java/com/github/victools/jsonschema/module/jackson/JacksonModuleTest.java

```diff
@@ -219,13 +219,36 @@ public void testDescriptionResolver(String fieldName, String expectedDescription
         Assert.assertEquals(expectedDescription, description);
     }
 
+    Object parametersForTestRequiredProperty() {
+        return new Object[][]{
+                {null,"requiredTrue", false},
+                {null,"requiredFalse", false},
+                {null,"requiredAbsent", false},
+                {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED,"requiredTrue", true},
+                {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED,"requiredFalse", false},
+                {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED,"requiredAbsent", false}
```

Comment:
inconsistent white-spaces; and we can add the `requiredDefault` case.
```suggestion
            {null, "requiredTrue", false},
            {null, "requiredFalse", false},
            {null, "requiredDefault", false},
            {null, "requiredAbsent", false},
            {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED, "requiredTrue", true},
            {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED, "requiredFalse", false},
            {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED, "requiredDefault", false},
            {JacksonOption.RESPECT_JSONPROPERTY_REQUIRED, "requiredAbsent", false}
```